depthai_sdk/examples/StereoComponent/stereo_record.py

```diff
@@ -0,0 +1,32 @@
+import cv2
+import depthai
+
+from depthai_sdk import OakCamera
+from depthai_sdk.visualize.configs import StereoColor
+
+
+def callback(msgs, visualizer):
+    for name, packet in msgs.items():
+        cv2.imshow(packet.name, packet.frame)
+
+
+with OakCamera(usbSpeed=depthai.UsbSpeed.HIGH) as oak:
+    color = oak.create_camera('color', resolution='1080p', fps=5)
+    stereo = oak.create_stereo('400p', fps=5)
+
+    stereo.configure_postprocessing(
+        colorize=StereoColor.RGB,
+        colormap=cv2.COLORMAP_JET,
+        wls_filter=True,
+        wls_lambda=8000,
+        wls_sigma=1.5
+    )
+
+    # Record RGB and disparity to records folder
+    # Record doesn't work with visualize so the config is ignored
+    oak.record([color.out.main, stereo.out.disparity], 'records')
+
+    # Record depth only
+    # oak.visualize(stereo.out.depth, record='depth.avi')
```

Comment:
Do common video players (eg VLC) work with int16 pixels?

Reply:
FFV1 codec supports 16-bit grayscale frames, I didn't test if it works well with video players (but VLC should support it). Now depth data is saved in binary format, I tried to use ffv1 codec with the current implementation but it didn't work. I'll spend more time on this later.